stock_available_mrp/models/product_product.py

```diff
@@ -94,11 +94,11 @@ def _compute_available_quantities_dict(self):
             # Need by product (same product can be in many BOM lines/levels)
             exploded_components = exploded_boms[product.id]
             component_needs = product._get_components_needs(
-                exploded_components)
+                exploded_components, True)
```

Comment:
pass the name of the kwarg:
```suggestion
                exploded_components, only_stockable=True)
```